cloud/aws/templates/aws_oidc/bin/aws_cli.py

```diff
@@ -14,51 +14,39 @@ def __init__(self, config: ConfigLoader):
 
     def is_secret_empty(self, secret_name: str) -> bool:
         res = self._call_cli(
-            [
-                'secretsmanager', 'get-secret-value',
-                f'--secret-id={secret_name}'
-            ])
+            f'secretsmanager get-secret-value --secret-id={secret_name}')
         return res['SecretString'].strip() == ''
 
-    def set_secret_value(self, secret_name: str, new_value: str) -> None:
+    def set_secret_value(self, secret_name: str, new_value: str):
         self._call_cli(
-            [
-                'secretsmanager', 'update-secret', f'--secret-id={secret_name}',
-                f'--secret-string={new_value}'
-            ])
+            f'secretsmanager update-secret --secret-id={secret_name} --secret-string={new_value}'
+        )
 
     def is_db_password_default(self, secret_name: str) -> bool:
         res = self._call_cli(
-            [
-                'secretsmanager', 'get-secret-value',
-                f'--secret-id={secret_name}'
-            ])
-        return res['SecretString'].startswith("default-")
+            f'secretsmanager get-secret-value --secret-id={secret_name}')
+        return res['SecretString'].startswith('default-')
 
     def get_current_user(self) -> str:
-        return self._call_cli(['sts', 'get-caller-identity'])['UserId']
+        return self._call_cli('sts get-caller-identity')['UserId']
```

Comment:
should this be just 'sts get-caller-identity UserId' ?

Reply:
Done.